src/distributions/mod.rs

```diff
@@ -13,20 +13,22 @@
 //! Distributions are stateless (i.e. immutable) objects controlling the
 //! production of values of some type `T` from a presumed uniform randomness
 //! source. These objects may have internal parameters set at contruction time
-//! (e.g. [`Range`], which has configurable bounds) or may have no internal
+//! (e.g. [`Uniform`], which has configurable bounds) or may have no internal
 //! parameters (e.g. [`Standard`]).
 //! 
 //! All distributions support the [`Distribution`] trait, and support usage
 //! via `distr.sample(&mut rng)` as well as via `rng.sample(distr)`.
 //! 
 //! [`Distribution`]: trait.Distribution.html
-//! [`Range`]: range/struct.Range.html
+//! [`Uniform`]: uniform/struct.Uniform.html
 //! [`Standard`]: struct.Standard.html
 
 use Rng;
 
 pub use self::other::Alphanumeric;
-pub use self::range::Range;
+pub use self::uniform::Uniform;
+#[deprecated(since="0.5.0", note="use Distribution instead")]
```

Comment:
Shouldn't this say "use Uniform instead"?

Reply:
Yes, I fixed it.